runconfig/parse.go

```diff
@@ -513,3 +505,31 @@ func ParseDevice(device string) (DeviceMapping, error) {
 	}
 	return deviceMapping, nil
 }
+
+// ParseVolumesIntoBinds parses any volumes passed by the client and moves any
+// volumes which are in Config.Volumes into HostConfig.Binds.
+func ParseVolumesIntoBinds(c *Config, hc *HostConfig) (*Config, *HostConfig, error) {
```

Comment:
Apart from the fact that now it doesn't actually matter if nothing is passed in Binds, can you give an example of what has in reality changed from a caller perspective? I haven't had to change any unit or integration test cases, they all operate as they did before. If there were was a breaking API change as such, it should have been caught. 

What would you asking me to change/do from your comment?